data/data/powervs/variables.tf

```diff
@@ -40,6 +40,12 @@ variable "cloud_instance_id" {
     default     = ""
 }
 
+variable "cloud_resource_group" {
+    type        = string
+    description = "The cloud instance resource group"
+    default     = ""
+}
+
```

Comment:
was this the file you wanted to rename?

Reply:
Yes, I believe this file should be named variables-powervs.tf, but didn't want to include it in this PR because this PR is the initial commit of service related stuff and a file rename may have been out of place.

Reply:
yeah I think this is what aws/gcp/etc does so I think it makes sense! +1 if you want to do it in this patch